internal/database/database.go

```diff
@@ -0,0 +1,287 @@
+package database
```

Comment:
How hard would it be to ask it to fetch sda-pipeline's database module? Then we would avoid config drift across services. Not sure this is a good idea.

Reply:
I did think about that at first, but viper seemed to be a bigger task to get working, so I started doing this without that, but we could definitely refactor the config logic and database module to those when going forward
Issues for these: #17 #18